overlay-scrollbars.user.css

```diff
@@ -1,15 +1,18 @@
 /* ==UserStyle==
-@name         overlay-scrollbars
-@version      2.0.0
+@name         Overlay scrollbars
```

Comment:
Would prefer if you keep my name, so the name matches the repo name.

Reply:
Sorry just looked better in Stylus :) restored in https://github.com/silverwind/overlay-scrollbars/pull/2/commits/c2917d7f356059e53f6da5c77050c3fabf0702d7